FirebaseAdmin/FirebaseAdmin.Tests/Auth/Providers/ProviderConfigManagerTest.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FirebaseAdmin.Tests;
+using FirebaseAdmin.Util;
+using Google.Apis.Auth.OAuth2;
+using Xunit;
+
+namespace FirebaseAdmin.Auth.Providers.Tests
+{
+    public class ProviderConfigManagerTest
+    {
+        private const string MockProjectId = "project1";
+
+        private const string ClientVersionHeader = "X-Client-Version";
+
+        private const string IdTooklitUrl = "https://identitytoolkit.googleapis.com/v2/projects/{0}";
```

Comment:
`IdToolkitUrl`?

Reply:
Actually I think this isn't used in this test

Reply:
Removed.